.mvn/wrapper/maven-wrapper.properties

```diff
@@ -1,2 +1,2 @@
-distributionUrl=https://ci-builds.apache.org/job/Maven/job/maven-box/job/maven/job/master/70/artifact/org/apache/maven/apache-maven/4.0.0-alpha-1-SNAPSHOT/apache-maven-4.0.0-alpha-1-SNAPSHOT-bin.zip
+distributionUrl=https://ci-builds.apache.org/job/Maven/job/maven-box/job/maven/job/master/lastSuccessfulBuild/artifact/org/apache/maven/apache-maven/4.0.0-alpha-1-SNAPSHOT/apache-maven-4.0.0-alpha-1-SNAPSHOT-bin.zip
```

Comment:
Neat!

Reply:
It's a temporary change, I need to find a better way to make our builds reproducible